app/src/main/java/com/endava/myendava/activities/SignInActivity.java

```diff
@@ -23,6 +30,9 @@
     @Inject
     MySharedPreferences mSharedPreferences;
 
+    @Inject
+    UsersViewModel mUsersViewModel;
```

Comment:
The UsersViewModel has nothing to do with the data from this screen. Create a new ViewModel that will be used in this screen.